# Medidas de Tendencia Central y Percentiles

## Objetivo:

Aplicar las medidas de tendencia central (media, mediana y moda) y medidas de posición no central (percentiles) en el análisis de datos, utilizando Python como herramienta de apoyo.

*Nota: Este laboratorio está diseñado para desarrollar razonamiento estadístico. Evita el uso de herramientas de IA para responder las preguntas a menos que esto sea explícitamente sugerido* 🤖. *En su lugar, enfócate en aplicar los conceptos revisados.*

## Actividad 1

La siguiente base de datos posee los registros de todas las calificaciones obtenidas por los alumnos del segundo semestre del 2019, jornada diurna en la asignatura de Álgebra.

Para responder las siguientes preguntas, descarga la base de datos **<a href="https://drive.google.com/uc?export=download&id=10tm66MrnBpkR1_aY10yxEAudwirzDcJK
">aquí</a>**.

1. ¿Cuál es la población en estudio?

La población del estudio son alumnos que durante el segundo semestre del 2019 cursaron en jornada diurna la asignatura de Álgebra.

2. ¿Cuál es la muestra en estudio?

In [1]:
import pandas as pd
base1=pd.read_csv('Actividad_1.csv')
base1.tail(2)

ID Alumno  Taller 1  Taller 2  Taller 3  Actividad en línea 1  \
514  160003661       7.0       7.0       7.0                   6.6   
515  160003452       7.0       7.0       6.8                   6.2   

     Actividad en línea 2  Actividad en línea 3  Prueba 1  Prueba 2  Prueba 3  \
514                   6.5                   5.7       6.4       6.6       6.2   
515                   4.9                   2.4       6.1       3.1       3.0   

     Nota Presentación  Nota Examen  Nota Final  
514                6.5          6.3         6.4  
515                4.5          4.9         4.7

In [3]:
base1.shape

(516, 13)

La muestra del estudio son 516 alumnos que durante el segundo semestre del 2019 cursaron en jornada diurna la asignatura de Álgebra.

3.  Identifique y clasifique las variables involucradas en el estudio.

Las variables involucradas son las calificaciones de:
- Taller 1
- Taller 2
- Taller 3
- Actividad en línea 1
- Actividad en línea 2
- Actividad en línea 3
- Prueba 1
- Prueba 2
- Prueba 3
- Nota de presentación
- Nota Examen 
- Nota final

Todas las variables involucradas son de tipo cuantitativa continua.

4. Calcule e interprete las medidas de tendencia central de las notas del examen.

In [13]:
examen=base1['Nota Examen']

media=examen.mean()
mediana=examen.median()
moda=examen.mode()[0]

print(f'La calificación promedio del curso obtenida en el examen es un {round(media,2)}')
print(f'El 50% de los estudiantes tiene una calificación menor o igual a {mediana:.2f}')
print(f'La calificación que más se repite entre las notas del examen es {moda}')

La calificación promedio del curso obtenida en el examen es un 5.05
El 50% de los estudiantes tiene una calificación menor o igual a 5.20
La calificación que más se repite entre las notas del examen es 5.3


5. ¿Entre qué valores se encuentra el 60% de las notas más bajas en la prueba 1?

In [14]:
prueba_1=base1['Prueba 1']
print(f'El 60% de las notas más bajas está entre un {prueba_1.min()} y {prueba_1.quantile(0.6)}')

El 60% de las notas más bajas está entre un 1.6 y 5.7


6.  La institución de educación creará cursos de reforzamiento para todos los estudiantes cuyas calificaciones en la Prueba 2, estén en el 25% más bajo. ¿Cuál es la nota más alta dentro de este grupo que va a reforzamiento?

In [15]:
print(f'La calificación más alta del grupo que irá a reforzamiento es {base1['Prueba 2'].quantile(0.25)}')

La calificación más alta del grupo que irá a reforzamiento es 4.4


7. ¿Cuál es la nota más baja entre el 30% de mejores notas del examen?

In [16]:
print(f'La calificación más baja entre el 30% con mejores notas del examen es {examen.quantile(0.7)}')

La calificación más baja entre el 30% con mejores notas del examen es 5.7


8. Si se desea reevaluar al 10% más bajo en el examen, ¿cuál es rango de notas que se debe considerar?

In [17]:
print(f'El 10% con las notas más bajas en el examen tiene entre {examen.min()} y {examen.quantile(0.1)}')

El 10% con las notas más bajas en el examen tiene entre 1.3 y 3.5


9. Si la institución va a premiar al 5% de las mejores notas finales, ¿cuál es la nota más baja registrada por ese grupo?

In [18]:
print(f'La nota más baja registrada del grupo premiado es un {base1['Nota Final'].quantile(0.95)}')

La nota más baja registrada del grupo premiado es un 6.5


## Actividad 2

Imagina que trabajas como analista de datos en una consultora de orientación vocacional que asesora a estudiantes de enseñanza media sobre qué carrera elegir. Recibes una base de datos con información de carreras técnicas y profesionales de distintas instituciones chilenas de los años 2024 y 2025. Tu misión es analizar los ingresos y empleabilidad de los titulados, y entregar conclusiones claras que los ayuden a tomar decisiones.

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. Identifica la población y muestra del estudio.

2. Clasifica las variables presentes en la base de datos.

3. ¿Cuál es el ingreso promedio de los titulados? ¿Y la mediana? ¿Qué puedes concluir de la diferencia entre ambos?

4. ¿Cuál es la moda de los ingresos? Interprétala.

5. Calcula los percentiles 25, 50 y 75 del ingreso de los titulados e interprétalos.

6. Utiliza la función *describe()* de Pandas para validar tus resultados.

7. Considera que tres personas desean estudiar Técnico en enfermería, pero cada una valora cosas distintas:
* Sofía busca maximizar su ingreso futuro.
* Diego prioriza encontrar trabajo rápidamente.
* Camila quiere un equilibrio entre ambas.

$\hspace{1cm}$Considerando la media como medida representativa, ¿qué tipo de institución recomendarías a cada uno y por qué?

## Actividad 3

Un grupo de estudiantes está evaluando si el nivel de acreditación de una institución podría impactar sus resultados laborales. Para responder esta pregunta, realiza el siguiente análisis:

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. Agrupa los datos por años de acreditación y calcula la media y la mediana del ingreso al 4to año de cada grupo.

2. ¿Existe una relación aparente entre la cantidad de años de acreditación y el ingreso promedio? ¿Qué patrón observas?

3. ¿Se observa una tendencia similar al analizar la empleabilidad al primer año?

4. Comparando las variables ingreso y empleabilidad, analiza cómo varían la media y la mediana en cada caso. ¿Qué conclusiones generales puedes extraer sobre la representatividad de estas medidas?

5. Teniendo en cuenta tu análisis sobre cómo varían los ingresos y la empleabilidad con los años de acreditación, ¿qué consejo le darías a un estudiante que va a elegir una institución basándose únicamente en el nivel de acreditación? Justifica tu recomendación.

## Actividad 4

Imagina que trabajas como analista del Ministerio de Educación. Te han solicitado caracterizar los resultados laborales de los titulados por área de formación utilizando una base de datos con información de carreras técnicas y profesionales de distintas instituciones chilenas de los años 2024 y 2025.

**Utiliza la siguiente base de datos para responder las actividades 2, 3 y 4.**

Descarga la base de datos **<a href="https://docs.google.com/spreadsheets/d/1O7lb2OqNCSnqJaswSNqmD4HuI0BWfcsP/export?format=xlsx
">aquí</a>**.

1. ¿Cuál Área se observa con mayor frecuencia entre los titulados?

2. ¿Entre qué sueldos se encuentran los egresados del 10% más alto al 4to año de titulación?

3. ¿Entre qué sueldos se encuentran los egresados del 17% más bajo al 4to año de titulación?

4. Agrupa los datos por Área y para cada grupo calcula las siguientes medidas del Ingreso al 4° año:
* Media
* Mediana
* Percentil 25
* Percentil 75

$\hspace{1cm}$Luego, construye un DataFrame que reúna estas cuatro columnas.

5. Interpreta las medidas calculadas en las áreas de Agropecuaria y Tecnología.

6. Al comparar la media con la mediana en el Área Salud se observa que la media es mayor. ¿Qué característica de los datos podría explicar esta diferencia?

7. Describe lo que ocurre con la media del ingreso al 4to año de titulación, según Área y Tipo de institución. ¿Qué conclusiones puedes extraer acerca del impacto del tipo de institución en el ingreso según área?

*Sugerencia: Puedes crear una tabla bivariada utilizando la función crosstab() de Pandas*.